rust/ql/lib/codeql/rust/elements/internal/CallableImpl.qll

```diff
@@ -17,5 +17,15 @@ module Impl {
    */
   class Callable extends Generated::Callable {
     override Param getParam(int index) { result = this.getParamList().getParam(index) }
+
+    /**
+     * Gets the self parameter of this callable, if it exists.
```

Comment:
Is there a reason why this is not "if any"?

Reply:
The doc was simply copied.